proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -466,7 +466,27 @@ message SyncActivityRequest {
     string last_worker_identity = 13;
     temporal.server.api.history.v1.VersionHistory version_history = 14;
     temporal.server.api.workflow.v1.BaseExecutionInfo base_execution_info = 15;
+}
 
+message SyncActivitiesRequest {
+    string namespace_id = 1;
+    string workflow_id = 2;
+    string run_id = 3;
+    repeated ActivitySyncInfo activities_info = 4;
+}
+message ActivitySyncInfo {
+    int64 version = 1;
+    int64 scheduled_event_id = 2;
+    google.protobuf.Timestamp scheduled_time = 3 [(gogoproto.stdtime) = true];
+    int64 started_event_id = 4;
+    google.protobuf.Timestamp started_time = 5 [(gogoproto.stdtime) = true];
+    google.protobuf.Timestamp last_heartbeat_time = 6 [(gogoproto.stdtime) = true];
+    temporal.api.common.v1.Payloads details = 7;
+    int32 attempt = 8;
+    temporal.api.failure.v1.Failure last_failure = 9;
+    string last_worker_identity = 10;
+    temporal.server.api.history.v1.VersionHistory version_history = 11;
+    temporal.server.api.workflow.v1.BaseExecutionInfo base_execution_info = 12;
```

Comment:
you do not need this right?

Reply:
I think we don't need this for now. I have not found any usage of this in current code. I copied this field from the original [request](https://github.com/temporalio/temporal/blob/d7fbf91cadb352da1042bc47b8cda4daaa0c8dcd/proto/internal/temporal/server/api/historyservice/v1/request_response.proto#L468).